src/cdk-experimental/testing/tests/harnesses/sub-component-harness.ts

```diff
@@ -6,12 +6,22 @@
  * found in the LICENSE file at https://angular.io/license
  */
 
-import {ComponentHarness} from '../../component-harness';
+import {ComponentHarness, HarnessPredicate} from '../../component-harness';
 import {TestElement} from '../../test-element';
 
+/** @dynamic */
 export class SubComponentHarness extends ComponentHarness {
   static readonly hostSelector = 'test-sub';
 
+  static with(options: {title?: string | RegExp, itemCount?: number} = {}) {
```

Comment:
Should we add an interface for supporting this `with` pattern to more formally codify it?

Reply:
I like this pattern and I think that Harness authors should follow it, but the actual options may differ based on the particular harness being implemented, and some harnesses might not even need a `with` method. 

Maybe we can just say its a recommended convention and do it in all of our component harnesses where it makes sense?

Reply:
That works for me